media/slides/index.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: media
+title: Media | Slides
+---          
+
+<!-- begin Slides -->
+<hr class="invisible">
+<div id="media-slides">
+
+<div class="small-12 columns">
+    <div class="media-block-grid">
+        <div class="media-block-grid-header small-12 medium-12 columns text-center">
+            <h2>Slides</h2>
+        </div>                
+        <div class="row small-up-1 medium-up-2 large-up-4">            
+            <div class="media-block column image-hover-wrapper">
+                <a href="https://downloads.openmicroscopy.org/presentations/2017/Cambridge-Training/Day1-Cambridge2017-Introduction.pptx"><img src="{{ site.baseurl }}/media/slides/thumbnails/2017-12-cambridge-training.png" alt="December 2017 Cambridge Training">
+                <span class="image-hover-wrapper-reveal">
+                    <p>Download slides<br><i class="fa fa-download" aria-hidden="true"></i></p>
+                </span>
+                </a>
+                <p class="text-center media-block-title">OMERO Users Training Day,<br>December 2017</p>
+            </div>
+            <div class="media-block column image-hover-wrapper">
+                <a href="https://downloads.openmicroscopy.org/presentations/2017/Cambridge-Training/Day1-Cambridge2017-Introduction.pdf"><img src="{{ site.baseurl }}/media/slides/thumbnails/2017-12-cambridge-training-pdf.png" alt="December 2017 Cambridge Training">
+                <span class="image-hover-wrapper-reveal">
+                    <p>Download PDF<br><i class="fa fa-download" aria-hidden="true"></i></p>
+                </span>
+                </a>
+                <p class="text-center media-block-title">OMERO Users Training Day,<br>December 2017 (PDF)</p>
```

Comment:
What's the contract here in terms of presentation format? The PDF format is specified but others are not.

Reply:
Ah, we should probably have the other slides labelled as .pptx
Don't know about the Reveal ones - given that they open in the browser, listing a format many not not be familiar with might discourage them from clicking through? Maybe we should use 'web-based' instead?

Reply:
:+1: for being vague about the exact web-based technology being used!

Reply:
Adding "web-based" makes the file names longer and may force them into 3 lines so I went with some simple iconography. Hope that's okay.

Reply:
Looks okay to me!